lcls-twincat-math/lcls_twincat_math/POUs/F_NormSqQuat.TcPOU

```diff
@@ -0,0 +1,16 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<TcPlcObject Version="1.1.0.1">
+  <POU Name="F_NormSqQuat" Id="{42312887-5f96-4969-87c2-7747d5e51a97}" SpecialFunc="None">
+    <Declaration><![CDATA[
+FUNCTION F_NormSqQuat : LREAL
```

Comment:
As I scroll through these I'm noticing that many of these routines are implemented multiple times the same way. For example, this function has the same effect as one of the function block methods.

In some cases may be cleaner to consolidate these a bit, using the functions as the source of the calculations and calling them in the methods instead of duplicating the implementation.